python_modules/dagster/dagster/_core/definitions/load_assets_from_modules.py

```diff
@@ -63,48 +149,12 @@ def assets_from_modules(
             A tuple containing a list of assets, a list of source assets, and a list of
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +144]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->lol i guess remove that<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/26484?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->